index.html

```diff
@@ -64,6 +64,188 @@ <h2 class="info-title">I'am a Software Developer</h2>
           </ul>
         </div>
       </section>
+      <section class="section-container">
+        <div class="container">
+          <h1 class="projects-title">My Recent Works</h1>
+
+          <article class="project-card">
+            <div class="card-header">
+              <img
+                src="/assets/Img-Placeholder.png"
+                alt="multi-post stories"
+                class="card-header-img"
+              />
```

Comment:
- kindly, make sure your image url is typed correctly if the file is in the same directory use`./` not `/`. 👍 👍